crates/re_space_view_spatial/src/ui_2d.rs

```diff
@@ -238,177 +165,136 @@ pub fn view_2d(
         draw_data,
     } = system_output;
 
-    // Save off the available_size since this is used for some of the layout updates later
-    let available_size = ui.available_size();
-
-    let scene_rect_accum = state.bounding_boxes.accumulated;
-    let scene_rect_accum = egui::Rect::from_min_max(
-        scene_rect_accum.min.truncate().to_array().into(),
-        scene_rect_accum.max.truncate().to_array().into(),
-    );
+    if ui.available_size().min_elem() <= 0.0 {
+        return Ok(());
+    }
 
-    // Determine the canvas which determines the extent of the explorable scene coordinates,
-    // and thus the size of the scroll area.
-    //
-    // TODO(andreas): We want to move away from the scroll area and instead work with open ended 2D scene coordinates!
-    // The term canvas might then refer to the area in scene coordinates visible at a given moment.
-    // Orthogonally, we'll want to visualize the resolution rectangle of the pinhole camera.
-    //
-    // For that we need to check if this is defined by a pinhole camera.
     // Note that we can't rely on the camera being part of scene.space_cameras since that requires
```

Comment:
`we'll want to visualize the resolution rectangle of the pinhole camera.` from the old todo note is still a thing we should add (not now, but warrants at least a note)